Eatery/Models/EaterySwipeData.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  EaterySwipeData.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 5/5/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+
+class EaterySwipeData {
+    
+    private var swipeDataByHours = [Int : [SwipeDataPoint]]()
+  
+    init(swipeDataPoints: [SwipeDataPoint]) {
+        mapSwipeDataPointToHours(swipeDataPoints: swipeDataPoints)
```

Comment:
For type safety, I think three things should happen: 

1. `private let swipeDataByHours: [Int: [SwipeDataPoint]]`
2. `mapSwipeDataPointToHours` functionality should be moved here, either by creating a temporary mutable `var swipeDataByHours` and later assigning it to `self.swipeDataByHours`, or by (3) below. 
3. Use functional methods (probably `reduce(into:)`) to replicate this functionality in one line.